### PR TITLE
fix: Update custom metadata token disclaimer

### PR DESCRIPTION
**Pull request by jacobcreech**
#### Problem

There have been quite a few reports of people creating tokens and then being confused by the disclaimer on their own token. 

#### Summary of Changes

Added some additional information + a CTA to verify based on feedback

### CODE REVIEW

**Comment by jstarry** [comment]
Can you add a screenshot?

**Comment by jacobcreech** [comment]
Screen Shot 2022-04-27 at 11 20 02 AM

File: explorer/src/pages/AccountDetailsPage.tsx [hunk 1]
```diff
@@ -209,8 +209,9 @@ export function AccountHeader({
       <div className="row align-items-end">
         {unverified && (
           <div className="alert alert-danger alert-scam" role="alert">
```
Comment (jstarry) [inline explorer/src/pages/AccountDetailsPage.tsx:211]: What do you think about changing this to "alert-warning"? It's not *as* scary as the red danger banner but still impossible to ignore.

Comment (jacobcreech) [reply explorer/src/pages/AccountDetailsPage.tsx:211]: Screen Shot 2022-04-27 at 11 26 22 AM
That pink/purple is just as hard to ignore. Let me update

Comment (jstarry) [reply explorer/src/pages/AccountDetailsPage.tsx:211]: cool looks good!

**Review by jstarry — APPROVED** [review]

**Comment by jstarry** [comment]
Having a CTA is much better, thanks!

**Pull request merged**